api/src/main/resources/marquez/db/migration/V68__add_jobs_tag_mapping.sql

```diff
@@ -0,0 +1,9 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+CREATE TABLE jobs_tag_mapping (
+  job_uuid UUID REFERENCES jobs(uuid),
+  tag_uuid     UUID REFERENCES tags(uuid),
+  tagged_at    TIMESTAMP NOT NULL,
```

Comment:
Use `TIMESTAMPTZ` to store timestamps as `UTC` and avoid handling time zones manually.